datalad/plugin/wtf.py

```diff
@@ -61,8 +87,21 @@ def dlplugin(dataset=None):
             result_filter=lambda x: x['action'] == 'metadata')
     if ds_meta:
         ds_meta = ds_meta['metadata']
-
     ui.message(report_template.format(
+        system='\n'.join(
+            '{}: {}'.format(*i) for i in (
+                ('OS          ', ' '.join([
+                    os.name,
+                    pl.system(),
+                    pl.release(),
+                    pl.version()]).rstrip()),
+                ('Distribution',
+                 ' '.join([_t2s(pl.dist()),
+                           _t2s(pl.mac_ver()),
+                           _t2s(pl.win32_ver())]).rstrip()))),
+        env='\n'.join(
+            '{}: {}'.format(k, v) for k, v in os.environ.iteritems()
```

Comment:
Use `six.iteritems` to avoid `AttributeError` in PY3.

Reply:
Thanks! Just used `.items()`, though...